crates/rattler_repodata_gateway/src/fetch/jlap/mod.rs

```diff
@@ -0,0 +1,853 @@
+//! # JLAP
+//!
+//! This module contains functions and data types for downloading and applying patches from JLAP
+//! files.
+//!
+//! JLAP files provide a way to incrementally retrieve and build the `repodata.json` files
+//! that conda compatible applications use to query conda packages. For more information about
+//! how this file format works, please read this CEP proposal:
+//!
+//! - <https://github.com/conda-incubator/ceps/pull/20/files>
+//!
+//! ## Example
+//!
+//! The recommended way to use this module is by using the JLAPManager struct. This struct is meant
+//! to act as a kind of "facade" object which orchestrates the underlying operations necessary
+//! to fetch JLAP data used to update our current `repodata.json` file.
```

Comment:
I think this comment is still out of date.

Reply:
heh, good catch! Will fix.